src/components/pod/RuntimeDetectionsCard.tsx

```diff
@@ -0,0 +1,131 @@
+import { Card, CardTitle, CardBody } from '@patternfly/react-core';
+import * as React from 'react';
+import SeverityLabel from '../shared/SeverityLabel';
+import FindingsList from '../shared/FindingsList';
+import { FalconClient } from 'crowdstrike-falcon';
+
+interface RuntimeDetectionsCardProps {
+  client: FalconClient;
+  pod: any;
+}
+
+export default function RuntimeDetectionsCard({ client, pod }: RuntimeDetectionsCardProps) {
+  const [promise, setPromise] = React.useState(null);
+
+  const sevs = ['unknown', 'informational', 'low', 'medium', 'high', 'critical'];
+  function sorter(a, b) {
+    return sevs.indexOf(b.severity) - sevs.indexOf(a.severity);
+  }
+
+  // This function is used to format the date in the format we want without installing any
+  // additional libraries
+  function formatDate(dateString: string): string {
+    const date = new Date(dateString);
+    const months = [
+      'Jan.',
+      'Feb.',
+      'Mar.',
+      'Apr.',
+      'May',
+      'Jun.',
+      'Jul.',
+      'Aug.',
+      'Sep.',
+      'Oct.',
+      'Nov.',
+      'Dec.',
+    ];
+
+    const month = months[date.getMonth()];
+    const day = date.getDate().toString().padStart(2, '0');
+    const year = date.getFullYear();
+    const hours = date.getHours().toString().padStart(2, '0');
+    const minutes = date.getMinutes().toString().padStart(2, '0');
+    const seconds = date.getSeconds().toString().padStart(2, '0');
+
+    return `${month} ${day}, ${year} ${hours}:${minutes}:${seconds}`;
+  }
+
+  const header = [
+    {
+      field: 'tacticAndTechnique',
+      width: 4,
+    },
+    {
+      field: 'detectTimestamp',
+      width: 1,
+    },
+    {
+      field: 'severity',
+      width: 1,
+    },
+  ] as {
+    // ensure width matches the component params, not generic number
+    field: string;
+    width?: 1 | 2 | 3 | 4 | 5;
+  }[];
+
+  const body = [
+    {
+      field: 'detectionDescription',
+      name: 'Description',
+    },
+    {
+      field: 'commandLine',
+      name: 'Command line',
+    },
+    {
+      field: 'imageDigest',
+      name: 'Image digest',
+    },
+  ];
```

Comment:
To more closely match the endpoint detections card let's do:

Header:
- detectionDescription
- severity
- detectTimestamp

Body:
- tacticAndTechnique
- actionTaken
- commandLine (format inside of `<pre>`)
- containerName (since this API actually gives us the container name back)